routes/api/v1/favorites.js

```diff
@@ -0,0 +1,29 @@
+//Requirements
+const env = require('dotenv').config();
+const fetch = require('node-fetch');
+const express = require('express');
+const app = express();
+const router= express.Router();
+const environment = process.env.NODE_ENV || 'development';
+const configuration = require('../../../knexfile')[environment];
+const database = require('knex')(configuration);
+const forecast = require('./forecast')
+//Functions
+  async function getUser(key){
+    let users = await database('users').where('api_key', key);
```

Comment:
maybe later you can create a User model and have all these db queries in there so you can call User.doStuff (or whatever)